server/findingCafe.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*
+
+# input : list organized by python dictionary
+# output : print (insert success or fail)
+
+import json
+import pymysql
+from datetime import datetime
+
+open_cafe_list = []
+
+# connect to DB
+def getConnection():
+    return pymysql.connect(host='localhost',
+    user='gaeul',
+    password='alpha',
+    db='CAFE',
+    charset='utf8mb4')
+
+# inquiry data from connected DB 
+def getTable():
+    conn = getConnection()
+    try:
+        with conn.cursor() as cursor :
+            cursor.execute("select * from CAFES")
+            cafe_list = cursor.fetchall()
+        return cafe_list
+    finally:
+        conn.close()
+
+# get information about date
+def getDate() :
+    now_day = datetime.today().weekday()
+    now = datetime.now()
+    now_hour = now.hour
+    now_minute = now.minute
+    return now_day, now_hour, now_minute #return type is tuple 
+
+# find out opening cafe
+def findOpenCafes(cafes):
+    now_day, now_hour, now_minute = getDate()
+    day_to_find = now_day + 7 # the number '7' is to match with attribute 
+    #loop for each cafe in cafe list
+    for cafe in cafes :
+        print (cafe)
```

Comment:
Is this for testing? If not, we shouldn't include this in production code

Reply:
Same applies to all print statements.

Reply:
I deleted all